packages/solana-client/src/create-and-send-spl-transaction.ts

```diff
@@ -47,7 +47,7 @@ export async function createAndSendSplTransaction(
     client.rpc.getLatestBlockhash().send(),
   ])
   const transactionMessage = createSplTransferTransaction({
-    amount: uiAmountToBigInt(amount, decimals).toString(),
+    amount: uiAmountToBigInt(amount, decimals),
```

Comment:
Cleanup of `createSplTransferTransaction` and pulling this out to the caller will happen in #508